bulletin/index.html

```diff
@@ -117,22 +117,23 @@ <h4 class="buarticle-subheading">MACILE Spanish currilum</h4>
 			</article>
 			<article class="buarticle">
 				<img src="/images/bulletin/busLeaders-2008-m.jpg">
-				<p>Visit from a Dominican business leader and a government official. They spoke about the job market and the government scholarship program. Students learned about opportunities and requirements for success.</p>
+				<p>Visit from a Dominican business leader and a government official. They spoke about the job markets and the government scholarships program. Students learned about opportunities and requirements for success.</p>
```

Comment:
You introduced various regressions. It needs to be: 

- the job market
- scholarship programs

Reply:
Done